src/hooks/useInitialValues.test.ts

```diff
@@ -287,9 +280,7 @@ describe('useInitialValues', () => {
       notes: '',
       screening_methods: [],
       date_of_birth: '',
-      date_of_birth_1: '',
       infant_name: '',
-      infant_name_1: '',
```

Comment:
This was actually a wrong assertion. It seemed to work because we were not properly cleaning up the tests.